### PR TITLE
Fork Merge

### PR DESCRIPTION
**Pull request by ArifPMan**

### CODE REVIEW

**Comment by ArifPMan** [comment]
Can you review this Hardik? Still in the making but it'll save time if you look at it.

**Comment by ArifPMan** [comment]
I have left two paras of introduction - you can retain the one you like. Please review and provide your f/b comments.

**Comment by ArifPMan** [comment]
Hi, Jaggi,

Please review this document and provide me your f/b comments.

File: v6/postman/collections/version_control_for_collections.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Version Control for Collections"
+page_id: "version_control_for_collections"
+warning: false
+---
+
+Postman's version control mechanism ensures you always have access to the most updated versions of your collections. It provides a versioning and conflict resolution system in a simple and most effective way. Different team members in a team workspace can access and work together on collections. It's easy to see what has changed, then merge the changes or even resolve the conflicts.   
+
+You can 'fork' a collection in a personal workspace or a team workspace that you're a part of. This creates a local copy. In Postman parlance, a forked collection is referred to as 'Source' because you'll use it to make changes/updates. Your base collection is the 'Destination' copy because Postman merges the changes to your base collection (destination).
```
Comment (ankitjaggi) [inline v6/postman/collections/version_control_for_collections.md]: It's not just a local copy. The collection will be synced across the entire team.

File: v6/postman/collections/version_control_for_collections.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Postman confirms the success and the screen in the dashboard changes as shown below:
+
+[![forked collection](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Fork_Merge3.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Fork_Merge3.png)
+
+You'll see the above collection in your team workspace 'Merge Conflict'. 
+
+
+#### Merging Changes
+
+It's easy to see changes and merge them. This section describes how to merge the changes made in your collections by your team members. In this example, let's consider two requests R1 and R2 (in ForkMerge collection) having 'Get' methods in 'My Workspace'. Now, we go and change these request methods to 'Del' in the ForkMerge collection in Merge Conflict workspace. To merge these changes, we go to the dashboard and click 'Merge'. Postman merges these changes successfully. This is illustrated clearly in the following screen. The 'Get' methods which are represented in green font initially, turn into red-colored-font 'Del' methods:
```
Comment (ankitjaggi) [inline v6/postman/collections/version_control_for_collections.md]: Why is the font color changing thing mentioned here? Doesn't make sense to me.

File: v6/postman/collections/version_control_for_collections.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+When you attempt to merge the changes by clicking 'Merge all changes', Postman provides you three options, as illustrated in the following screen:
+
+[![merge all changes](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Fork_Merge_all_changes.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Fork_Merge_all_changes.png)
+
+Choose either of the options and click 'Merge'. 
+
+Let's say you forked a collection and made updates to the forked collection. If there are changes in the base collection, Postman notifies this conflict and allows you to resolve. The following Gif file illustrates this:
+
+[![merging conflict](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Fork_Merge5.gif)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Fork_Merge5.gif)
```
Comment (ankitjaggi) [inline v6/postman/collections/version_control_for_collections.md]: This gif does not show a conflict issue. This happens when your fork (source) collection goes out of sync with the base (destination) collection.

Comment (ArifPMan) [reply v6/postman/collections/version_control_for_collections.md]: Please review and approve from your side Jaggi. All of your comments are incorporated.

File: v6/postman/collections/version_control_for_collections.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Version Control for Collections"
+page_id: "version_control_for_collections"
+warning: false
+---
+
+Version control for collections in Postman allows multiple users to work on the same collection. It ensures you always have access to the updated versions  of collections. It's easy to see what has changed, pull the changes, merge them and even resolve the conflicts.
```
Comment (vkaegis) [inline v6/postman/collections/version_control_for_collections.md]: "Allow user to work on the same collection" might be a bit misleading. A fork is a different collection. 

Version control provides more control over collaborative flows by allowing users to work on different "forks" of the same collection. They can later merge and pull changes between these forks to keep the collections up to date with each other.

File: v6/postman/collections/version_control_for_collections.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+[![fork collection](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Forking_a_Collection.gif)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Forking_a_Collection.gif)
+
+
+#### Pulling Changes 
+
+After forking a collection, the base collection may be updated by others. If that happens, your forked copy will no longer be the same. In such circumstances, Postman notifies you about this change and allows you to pull these changes. As discussed in the previous section, the HealthAPICheck collection is forked in Team-Cobras. This collection is now updated in the personal workspace making the forked and base collection out of sync. In this case, you may want to pull the changes. The following screen illustrates this:
```
Comment (vkaegis) [inline v6/postman/collections/version_control_for_collections.md]: "Parent" collection, not base. 

"If that happens, your FORK (not forked copy) will no longer be the same as the parent".

GIF is okay again. A bit long. In the future, we'll be adding an explicit context menu (...) option to pull changes directly without going into the merge flow.

File: v6/postman/collections/version_control_for_collections.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+[![pull changes](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Fork_Pull_Changes.gif)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Fork_Pull_Changes.gif)
+
+
+#### Merging Changes
+
+It's easy to view changes, review them and then merge. This section describes how to merge the changes made in your collections by others.
```
Comment (vkaegis) [inline v6/postman/collections/version_control_for_collections.md]: "merge changes made in your FORK with the parent collection."

File: v6/postman/collections/version_control_for_collections.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+Let's change the name of the forked collection in the above example to HealthAPICheck - Forked Collection which is part of Team-Cobras workspace. 
+
+The base collection - HealthAPICheck has four request R1, R2, R3, and R4. All of them are 'Get' requests. A copy of this collection now exists in Team-Cobras workspace. All four requests are updated in Team-Cobras workspace and those changes are merged in the base collection. The following screen illustrates this:
+
+[![merging changes](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Forking_n_Merging.gif)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Forking_n_Merging.gif)
+
+When you attempt to merge the changes by clicking 'Merge all changes', Postman provides the following three options, as illustrated:
```
Comment (vkaegis) [inline v6/postman/collections/version_control_for_collections.md]: We should probably expand on the three merge options

File: v6/postman/collections/version_control_for_collections.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+[![merge all changes](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Merge_All_Changes.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Merge_All_Changes.png)
+
+Choose either of the options and click 'Merge'. 
+
+#### Resolving Conflicts 
+
+This section illustrates how you can resolve conflicts when changes happen in more than a single workspace. The Request R2 gets changed in both the workspaces - personal and Team-Cobras. Postman notifies the conflict and prompts you to resolve them. The following screen illustrates this:
```
Comment (vkaegis) [inline v6/postman/collections/version_control_for_collections.md]: Would be good to explain when and how conflicts might arise. And also explain in detail the steps taken to resolve all conflicts, and how and when the resolution happens.

File: v6/postman/collections/version_control_for_collections.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Version Control for Collections"
+page_id: "version_control_for_collections"
+warning: false
+---
+
+Version control for collections in Postman allows multiple users to work on the same collection. It ensures you always have access to the updated versions  of collections. It's easy to see what has changed, pull the changes, merge them and even resolve the conflicts.   
+
+You can 'fork' a collection in a personal or a team workspace. To create a copy (fork) of a collection, either the workspace must belong to you or you should be a member of a team workspace. In Postman, a forked collection is referred to as 'Source' because you'll use it to make changes/updates. Your base collection is the 'Destination' copy because Postman merges your changes to the base collection (destination).
```
[the remark this answers is not in the document]
Comment (ArifPMan) [reply v6/postman/collections/version_control_for_collections.md]: Addressed

File: v6/postman/collections/version_control_for_collections.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+This section describes the following topics:
+
+* Forking a Collection
+* Pulling Changes 
+* Merging Changes
+* Resolving Conflicts
+
+#### Forking a Collection
+
+You can fork a collection from the Postman app as well as Postman dashboard. To fork a collection, go to your Postman app, click (...) adjacent to a collection and select 'Create a Fork'. Specify a name for this copy, select a workspace and click 'Fork Collection'. Let's consider an example where HealthAPICheck collection belongs to a personal workspace. A copy of this is created (forked) in Team-Cobras workspace where there are no collections. The following screen illustrates this:
```
[the remark this answers is not in the document]
Comment (ArifPMan) [reply v6/postman/collections/version_control_for_collections.md]: Addressed

File: v6/postman/collections/version_control_for_collections.md (hunk 5 above)
Comment (ArifPMan) [reply v6/postman/collections/version_control_for_collections.md]: Addressed

File: v6/postman/collections/version_control_for_collections.md (hunk 6 above)
Comment (ArifPMan) [reply v6/postman/collections/version_control_for_collections.md]: Addressed

File: v6/postman/collections/version_control_for_collections.md (hunk 7 above)
Comment (ArifPMan) [reply v6/postman/collections/version_control_for_collections.md]: Done!

File: v6/postman/collections/version_control_for_collections.md (hunk 8 above)
Comment (ArifPMan) [reply v6/postman/collections/version_control_for_collections.md]: Need your inputs, Vineet.

File: v6/postman/collections/version_control_for_collections.md (hunk 4 above)
Comment (ArifPMan) [reply v6/postman/collections/version_control_for_collections.md]: Addressed!

**Comment by ArifPMan** [comment]
Hey, Numaan,

Please review and provide your f/b comments..

File: v6/postman/collections/version_control_for_collections.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+
+[![merging changes](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Forking_n_Merging.gif)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Forking_n_Merging.gif)
+
+When you attempt to merge the changes by clicking 'Merge all changes', Postman provides the following three options, as illustrated:
+
+[![merge all changes](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Merge_All_Changes.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Merge_All_Changes.png)
+
+The 'Confirm merge' dialog offers the following choices:
+
+* **Merge changes** changes made in your fork will be merged into the parent.
```
Comment (numaanashraf) [inline v6/postman/collections/version_control_for_collections.md]: Can we add a delimited between the choice and the explanation? Like

**Merge changes** - changes made in your fork will be merged into the parent.

File: v6/postman/collections/version_control_for_collections.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+
+* **Merge changes** changes made in your fork will be merged into the parent. 
+*  **Merge changes and update source** changes made in your fork will be merged into the parent and the fork is updated. 
+*  **Merge changes and delete source** changes made in your fork will be merged into the parent and the fork is deleted.
+
+Choose either of the options and click 'Merge'. 
+
+#### Resolving conflicts 
+
+A conflict occurs in Postman when two or more users make different changes to the same request simultaneously. In such a scenario, Postman will not be able to decide which of these changes to retain.
```
Comment (numaanashraf) [inline v6/postman/collections/version_control_for_collections.md]: conflicts can occur on folders and examples too

**Pull request merged**